frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/ticketGroups.component.tsx

```diff
@@ -20,7 +20,7 @@ import { Group, GroupOverride, Viewpoint, ViewpointState } from '@/v5/store/tick
 import { useEffect, useState } from 'react';
```

Comment:
From the tickets list, when I select a ticket without expanding it (i.e. I'm still in the list view), I can't see the overrides and the transformations reflected on the model

**ticket is selected**
![image](https://github.com/3drepo/3drepo.io/assets/43547703/f3f0f4df-aa37-4389-a6c8-6613eabefa1e)

**ticket is expanded**
![image](https://github.com/3drepo/3drepo.io/assets/43547703/3c6d7ac2-336c-4108-a2c5-ea8a88682797)

Reply:
Duplicated comment? @Amantini1997 ?